CHANGELOG.rst

```diff
@@ -2,6 +2,11 @@
 Changelog for package moveit_msgs
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+2.2.0 (2021-12-10)
+------------------
+* Add hybrid planning action definitons (`#135 <https://github.com/ros-planning/moveit_msgs/issues/135>`_)
```

Comment:
It's strange that this URL doesn't work. I guess it was auto-generated? It's a pull request, not an issue

Should be

https://github.com/ros-planning/moveit_msgs/pull/135

Reply:
I think github must have changed :(

These generated links used to work.

Reply:
Here is the code that generates this: https://github.com/ros-infrastructure/catkin_pkg/blob/e5f553826cff83b18cf2d438c7df6c13b3c95c4a/src/catkin_pkg/changelog_generator_vcs.py#L261

All the other links are also broken in this changelog.  /issues/ used to point to pull requests, github must have changed.

Reply:
Created this issue: https://github.com/ros-infrastructure/catkin_pkg/issues/319

Reply:
Shall we merge it anyway? It's no worse than any of the rest of the Changelogs (and really, it's not a huge issue anyway).

Reply:
I think so... I don't think we should block our releases on this issue.  No harm in just fixing this later once the ros-infrastructure catches up.

Reply:
It works for me, it actually redirects to pull link.

Reply:
It now seems to be fixed.  Maybe github itself was broken?